### DISCUSSION TITLE
EL4 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#220] [Show and tell]
https://teambasedinquirylearning.github.io/precalculus/2024/EL4.html

Author: @AbbyANoble 
Reviewer: @tdegeorge

**Comment by tdegeorge** [comment]
Activity 5.4.1 - the "-" for "x-values" is longer than the other dash. Just looks a little odd when reading :)

Extra period after the answer choice in parts c and d.

Do we want to specifically state to plug in x-values into the function g(x) since they would have focused on the exponential form in previous parts? Or is the intent to have them look at the graph because one of the choices specifically says that the graph is not defined when x approaches infinity/negative infinity?

Part f: Could we maybe simplify the choices even further and just write some asymptotes as answers? Like y=0, x=0, etc.? In the rationals section, they should already know about the asymptotes in equation form. And if they believe there are more than one, they can pick more than one answer choice. If not, I would prefer A-E as I agree that my students would think the same way about there being a horizontal asymptote near 3 or 4.

Part h: shouldn't the increasing be (0, infinity)?

**Reply by AbbyANoble** [reply]
By extra period in (c) and (d), do you mean the one that's there? I don't see an extra. I put one there because they were complete sentences.

**Reply by AbbyANoble** [reply]
Re: plugging in x-values -- I was thinking to have those two questions the same to get them to remember that negative numbers aren't in the domain. I also meant to put one in this spot about what's happening around zero. Do you think that would be helpful too? Like, "What seems to be happening with the graph at x values near zero?" And answer choices similar to the others. Like "As x->0 from the positive direction, y->- infinity" and "The graph isn't defined as x->0 from the negative direction"

**Reply by AbbyANoble** [reply]
Re: asymptotes --- I had a similar question in EL2 and @kathypinzon went with the F-I style. 😆 See the first comment in #218. And then how I made updates based on her suggestion here: [Activity 5.2.1(e)](https://teambasedinquirylearning.github.io/precalculus/2024/EL2.html#activity-exp-graphs-intro-6)

**Reply by AbbyANoble** [reply]
part h -- fixed! thanks!

**Reply by tdegeorge** [reply]
> By extra period in (c) and (d), do you mean the one that's there? I don't see an extra. I put one there because they were complete sentences.

I just meant that when you put the answer in, you put D and then a period after that. In the previous answer choices, it just has the letter of the answer choice. Not a big deal - just noticed the difference.

**Reply by tdegeorge** [reply]
> Re: plugging in x-values -- I was thinking to have those two questions the same to get them to remember that negative numbers aren't in the domain. I also meant to put one in this spot about what's happening around zero. Do you think that would be helpful too? Like, "What seems to be happening with the graph at x values near zero?" And answer choices similar to the others. Like "As x->0 from the positive direction, y->- infinity" and "The graph isn't defined as x->0 from the negative direction"

Ok, I see what you are doing there.

I like the idea of what's happening around zero.

**Reply by tdegeorge** [reply]
> Activity 5.2.1(e)

I'm not opposed to that style - I was just thinking that technically students should already know about asymptotes and should also be able to give the equation for each kind. I still find students mixing them up.

I guess Kathy and I will talk about it at the meeting tomorrow and let you know!

**Reply by AbbyANoble** [reply]
> I just meant that when you put the answer in, you put D and then a period after that. In the previous answer choices, it just has the letter of the answer choice. Not a big deal - just noticed the difference.

Looking back, I am inconsistent about this. Haha. Sometimes I put a period and sometimes I don't. 🤦‍♀️

**Reply by kathypinzon** [reply]
I guess we should add that as an issue for editathon - check that all formatting for <answer>s is the same.

**Reply by AbbyANoble** [reply]
> I'm not opposed to that style - I was just thinking that technically students should already know about asymptotes and should also be able to give the equation for each kind. I still find students mixing them up.
> 
> I guess Kathy and I will talk about it at the meeting tomorrow and let you know!

I've updated it (and will submit the pull request soon). Let me know what you think! It has them think about if there are asymptotes, then pick them out.

**Comment by tdegeorge** [comment]
Remark 5.4.3 - there needs to be a space between "b not equal to 1" and "has"

**Comment by tdegeorge** [comment]
Activity 5.4.5 - the answers for part b don't match with the log equations given.

**Reply by AbbyANoble** [reply]
@kathypinzon had suggestions to add to the problem like this in EL2. See parts c-f of [Activity 5.2.8](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL2.html#activity-exp-graphs-match). Should I add similar parts in here?

**Reply by tdegeorge** [reply]
I like that idea!

**Reply by AbbyANoble** [reply]
Updated to mirror [Activity 5.2.8](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL2.html#activity-exp-graphs-match)!

**Comment by tdegeorge** [comment]
Activity 5.4.6 - do we also want to include graphing the asymptote? I see you have it in your answer, so was not sure if we wanted to be explicit about that too

**Reply by AbbyANoble** [reply]
@kathypinzon said the same for the EL2 one. Do you like how these are worded? [Activity 5.2.9](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL2.html#activity-exp-e)

**Reply by tdegeorge** [reply]
I think that would be good too

**Comment by tdegeorge** [comment]
Activity 5.4.7 - answers do not have domain and range included.

**Reply by AbbyANoble** [reply]
Should I also include the equation of the asymptote? To mirror [Activity 5.2.10](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL2.html#activity-exp-graphs-fluency)?

**Reply by tdegeorge** [reply]
Yep, I agree with that.

**Comment by AbbyANoble** [comment]
Ok, I've sent in the pull request for this one now!
